lti/sentry.py

```diff
@@ -0,0 +1,19 @@
+"""Report exceptions to Sentry."""
+
+import os
+
+import raven
+
+
+def get_raven_client(request):
+    """Return the Raven client for reporting crashes to Sentry."""
+    client = request.registry["raven.client"]
```

Comment:
Yep, that's fine. Maybe it's something to keep in mind for later, maybe something like the class id / student id, anything like that.